src/app/[lang]/(protected)/chat/[chatId]/page.tsx

```diff
@@ -1,5 +1,53 @@
 import ChatWrapper from "./chatWrapper";
+import { getJwtAccessToken } from "../../regenerateAccessToken";
+import { env } from "@/env.mjs";
 
-export default async function ChatID({ params }: { params: { lang: Lang } }) {
-  return <ChatWrapper />;
+export type Message = {
+  uuid: string;
+  content: string;
+  user: {
+    uiid: string;
```

Comment:
Data that we receive have `uuid` instead of `uiid`

```suggestion
    uuid: string;
```